document-sandbox-samples/communication-iframe-documentSandbox/src/manifest.json

```diff
@@ -17,7 +17,7 @@
             "type": "panel",
             "id": "panel1",
             "main": "index.html",
-            "script": "code.js"
+            "documentSandbox": "code.js"
```

Comment:
CLI is released with this support so adding entryPoints.documentSandbox in manifest again